src/bin/import.js

```diff
@@ -40,7 +39,7 @@ var importProject = function (Storage, gmeConfig, projectId, jsonProject, branch
         };
 
 
-    storage = new Storage({globConf: gmeConfig, log: logger.fork('storage')});
+    storage = new Storage({globConf: _gmeConfig, logger: logger.fork('storage')});
```

Comment:
why the gmeConfig changed to _gmeConfig? (I mean the definition of it doesn't changed)
the same goes for line 52.